src/components/ha-form/types.ts

```diff
@@ -71,6 +71,7 @@ export interface HaFormFloatSchema extends HaFormBaseSchema {
 export interface HaFormStringSchema extends HaFormBaseSchema {
   type: "string";
   format?: string;
+  autofill?: string;
```

Comment:
Why not naming this variable `autocomplete` as well? IMO this will reduce confusion. 

Additionally, what about adding some documentation or adding a link to https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete to make it easier for devs to find out what values are allowed here.

Reply:
I can add a comment but the naming is consistent with previous changes and done purposefully to name the functionality (i.e. separate from `aria-autocomplete`)

Reply:
> I can add a comment but the naming is consistent with previous changes and done purposefully to name the functionality (i.e. separate from `aria-autocomplete`)

I am not that familiar with the HA source code, so I would say you have your reasons. However, I find it confusing to call a variable `autofill` that ends up filling the `autocomplete` attribute of an input field.

If the `autofill` is kept, then it makes even more sense to add a comment to clear up misunderstandings.